src/tests/example.spec.js

```diff
@@ -114,4 +114,56 @@ test.describe('Saucedemo app basic tests', () => {
 
         expect(actualSortedResultsZa).toEqual(expectedSortedResultZa);
     });
+
```

Comment:
In some tests, login is performed through beforeEach. However, it is better to do this with a separate loginAsStandardUser function, for example. This will improve readability and reduce duplication.